test/Profiler/coverage_relative_path_exec.swift

```diff
@@ -0,0 +1,25 @@
+// REQUIRES: profile_runtime
+// REQUIRES: executable_test
+// REQUIRES: OS=macosx
+
+// This test is to make sure llvm-cov can deal with a coverage-prefix-map.
+
+// To make sure this test is resilient to directory changes, we create nested directories inside of the
+// temporary test directory and assert those exist, or don't exist, in the emitted ir
```

Comment:
We're not actually checking for the directory are we? Just that the function exists in the output of `llvm-cov show`.

Reply:
llvm-cov barfs if it can't find the file, but yeah probably worth adding a check for the file

Reply:
Oh, I realize now you are talking about the comment, I'll update that